src/ui/map.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import L from "leaflet";
+require('leaflet/dist/leaflet.css')
```

Comment:
Why move this from index.html?

Reply:
No good reason, just got a bit overzealous when pruning things for there, think i may've also been confused that it referenced node_modules when they werent committed